server/sensorAlerts.js

```diff
@@ -29,6 +35,21 @@ async function handleAlert(location, alertType) {
     const currentTime = await db.getCurrentTime(pgClient)
     const client = location.client
 
+    // default to this sensor not being resettable
+    let isResettable = false
+
+    const parsedAlertData = JSON.parse(alertData)
+
+    // Sensors with version <= 10.7.0 will send alert data that is a string with no particularly useful information.
+    // This check is a safeguard in the case that an old sensor generates an alert before upgrading to a version that sends useful alert data.
```

Comment:
Wow! Great catch! Did you find this out by testing it or by thinking it through?

This would be a really good unit test to see that it behaves correctly with the old `data="stillness alert!!!"` string.